guides/server/telemetry.md

```diff
@@ -147,6 +143,50 @@ LiveView currently exposes the following [`telemetry`](https://hexdocs.pm/teleme
             event: String.t(),
             params: unsigned_params
           }
+          
+  * `[:phoenix, :live_view, :handle_info, :start]` - Dispatched by a `Phoenix.LiveView`
```

Comment:
I think we can avoid instrumenting handle_info since it's entirely user land, and just like GenServer doesn't instrument it for you, we can follow the same here